packages/wonder-blocks-search-field/src/components/search-field.tsx

```diff
@@ -167,8 +195,14 @@ const SearchField: React.ForwardRefExoticComponent<
                 <View onClick={onClick} style={[styles.inputContainer, style]}>
                     <PhosphorIcon
                         icon={magnifyingGlassIcon}
-                        size="medium"
-                        color={color.offBlack64}
+                        size="small"
```

Comment:
Applying some styling updates to make the search field look more like the Figma component! It also addresses the disabled and light states

Before (left) and After (right):
![image](https://github.com/user-attachments/assets/9800d627-7eff-4ae2-954f-9aaee2cd2263)